extension/src/commands/util.ts

```diff
@@ -18,3 +20,40 @@ export const showSetupOrExecuteCommand =
 
     return callback(context)
   }
+
+const enum PLOT_TYPE {
+  CUSTOM = 'custom',
+  TOP_LEVEL = 'top-level'
+}
+
+export const addPlotCommand = async (context: Context) => {
+  const result: PLOT_TYPE | undefined = await quickPickValue(
+    [
+      {
+        description: 'Create a dvc.yaml plot based off a chosen data file',
+        label: 'Top-Level',
+        value: PLOT_TYPE.TOP_LEVEL
+      },
+      {
+        description:
+          'Create an extension-only plot based off a chosen metric and param',
```

Comment:
[C] `Create an extension-only plot by selecting one metric and one param from the experiments table`

Reply:
Both of these are suggestions and can be dismissed/improved. What I am trying to do is standardise on simple terminology that helps users to connect the dots between concepts.

Reply:
[C] I think we need to revisit the `Custom` name now (not in this PR but we should discuss/consider updating).